pcmanfm/desktopwindow.cpp

```diff
@@ -948,7 +948,7 @@ bool DesktopWindow::eventFilter(QObject* watched, QEvent* event) {
             break;
         }
     }
-    return false;
+    return Fm::FolderView::eventFilter(watched, event);
```

Comment:
I'm not fond of hairsplitting; I enjoy coding because it's fun. Sometimes I feel that you make no hierarchy of things according to their relative importance in your mind. That's OK but I *do*. Why should I even think about adding a `View::eventFilter()` and override it while I can just change a line to fix a bug?! That's not my style.